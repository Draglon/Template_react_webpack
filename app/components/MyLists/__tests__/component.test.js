import React from 'react';

import MyListsComponent from '../component';

describe('<MyListsComponent />', () => {
  const props = {
    myLists: {
      results: [1, 2, 3],
      page: 1,
      totalPages: 10,
    },
    page: () => ({ page: 1 }),
  };

  const component = shallow(<MyListsComponent {...props} />);

  it('should match snapshot', () => {
    expect(component).toMatchSnapshot();
  });
});
