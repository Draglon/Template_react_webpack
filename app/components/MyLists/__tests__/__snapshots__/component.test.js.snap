// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MyListsComponent /> should match snapshot 1`] = `
<Fragment>
  <MyListsTitleComponent />
  <Connect(ListsContainer)
    lists={
      Array [
        1,
        2,
        3,
      ]
    }
  />
  <PaginationComponent
    currentPage={1}
    page={[Function]}
    totalPages={10}
  />
</Fragment>
`;
